0002-data-collection-extension/README.md

```diff
@@ -97,97 +102,21 @@ The payload MUST be a UTF-8 encoded JSON object with the following structure:
   "$schema": "http://json-schema.org/schema#",
   "title": "2/DCX request schema",
 
-  "oneOf": [
-    {
-      "type": "object",
-      "properties": {
-        "id": {
-          "type": "number",
-          "multipleOf": 1.0
-        },
-        "entries": {
-          "type": "array"
-        }
-      },
-      "required": [ "entries" ],
-      "additionalProperties": false
-    },
-    {
-      "type": "array"
-    }
-  ]
+  "type": "array"
 }
 ```
-where:
-- `id` (optional) is the batch ID.
-Should be integer number.
-- `entries` (required) is an array of data entries.
-Each entry can be of any JSON type.
+where each element of an array represents a single record.
 
 Example:
 ```json
-{
-  "id": 42,
-  "entries": [
-    { "key": "value" },
-    15,
-    [ "an", "array", 13 ]
-  ]
-}
-```
-
-If the `id` field is present, the server MUST respond with a processing confirmation response.
-
->**NOTE:** MQTT packet ID cannot be used.
->If used, it would not allow working via gateways as a gateway can change MQTT packet ID.
-
-If a batch does not specify the ID, the request degrades into a single-field JSON object.
-```json
-{
-  "entries": [ 15 ]
-}
-```
-
-In that case, the client MAY omit the `{"entries": }` part.
-```json
 [
   { "key": "value" },
   15,
   [ "an", "array", 13 ]
 ]
 ```
 
-##### Response
-
-JSON Schema for responses is defined in the [response.schema.json](./response.schema.json) file.
+#### Response
+A processing confirmation response MUST follow the error response format as defined per 1/KP.
```

Comment:
I think it's better to add link here to corresponding section. + I think it makes no sense to return `statusCode` and `reasonPhrase` in case of successful processing